### ISSUE TITLE
Design: Model Interoperability

### ISSUE THREAD

**Issue by nstogner** [substratusai/substratus#116]
How can we make all Model loader containers, Model trainer containers, and Server containers interoperate when they might be storing artifacts in various formats?

- Huggingface libraries store models one way
- Using raw pytorch models can be saved another way
- Tensorflow ...

Is ONNX (#67) the answer? If so, should Models be saved in the format native to their container's codebase and then converted to/from ONNX? Or should they just be stored in solely using the ONNX format?

How will a Substratus Model object represent its format?

Can a Model's format be determined from source / from the container image? Perhaps this info could be carrier in [container labels](https://github.com/substratusai/substratus/issues/67)?
